app/models/ebean/AlertEtags.java

```diff
@@ -53,7 +55,8 @@
      * @param organization the organization
      */
     public static void incrementEtag(final Organization organization) {
-        AlertEtags etag = FINDER.setForUpdate(true)
+        AlertEtags etag = FINDER.query()
```

Comment:
final?

Reply:
Nope, it can also get set a few lines down.  Also, I'm just here to update Play.